client/nats_test.go

```diff
@@ -0,0 +1,71 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/nats-io/nats-server/v2/server"
+	natsserver "github.com/nats-io/nats-server/v2/test"
+	"github.com/nats-io/nats.go"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/valocode/bubbly/agent/component"
+	"github.com/valocode/bubbly/api/core"
+	"github.com/valocode/bubbly/config"
+	"github.com/valocode/bubbly/env"
+)
+
+const TEST_PORT = 8131
```

Comment:
If this value is not completely arbitrary, I'd add a comment explaining why it was picked. In fact, if it's an arbitrary value, I'd add a comment saying so 🙃